### ISSUE TITLE
Question on overfitting

### ISSUE THREAD

**Issue by Contextualist** [as-ideas/DeepPhonemizer#18]
When I am training a forward transformer model, I observed that after the validation loss started to rise, PER and WER kept descending.

![validate](https://user-images.githubusercontent.com/16000021/158889755-2961ae4e-4fab-47f3-b5d9-0d22d5593e6b.png)

My training config is based on the example forward transformer config, with the `phoneme_symbols` modified (Phonemes are in ARPABET and vowels have stress marks) and `dropout` set to 0.3.

Should I keep training or should I use the model with the lowest validation loss? Or any other suggestion?

**Comment by cschaefer26** [comment]
Hi, yeah definitely train until the WER/PER are saturated. In my experience it is common for the validation loss to go up while other metrics tend to improve (also for language generation tasks etc.).

**Comment by Contextualist** [comment]
Got it, thanks for your explanation, @cschaefer26!
